app/views/projects/_statistics.html.erb

```diff
@@ -1,115 +1,93 @@
-      </td>
-      <td>
-        <strong><%= timeago_tag(@project.first_release.try(:published_at).presence || @project.repository.try(:created_at).presence || @project.created_at) %></strong>
-      </td>
-    </tr>
+    </dd>
+    <dt class='col-sm-8'>
+      Latest <%= @project.release_or_tag.singularize %>
+    </dt>
+    <dd class='col-sm-4'>
+      <%= timeago_tag(@project.latest_release.try(:published_at).presence || @project.updated_at) %>
+    </dd>
+    <dt class='col-sm-8'>
+      First <%= @project.release_or_tag.singularize %>
+    </dt>
+    <dd class='col-sm-4'>
+      <%= timeago_tag(@project.first_release.try(:published_at).presence || @project.repository.try(:created_at).presence || @project.created_at) %>
+    </dd>
     <% if @project.repository.present? %>
-    <tr>
-      <td>
+      <dt class='col-sm-8'>
         Stars
-      </td>
-      <td>
-        <strong><%= link_to number_to_human(@project.repository.stargazers_count), @project.repository.stargazers_url %></strong>
-      </td>
-    </tr>
-    <tr>
-      <td>
+      </dt>
+      <dd class='col-sm-4'>
+        <%= link_to number_to_human(@project.repository.stargazers_count), @project.repository.stargazers_url %>
+      </dd>
+      <dt class='col-sm-8'>
         Forks
-      </td>
-      <td>
-        <strong><%= link_to number_to_human(@project.repository.forks_count), @project.repository.forks_url %></strong>
-      </td>
-    </tr>
-    <% if @project.repository.subscribers_count.present? %>
-      <tr>
-        <td>
+      </dt>
+      <dd class='col-sm-4'>
+        <%= link_to number_to_human(@project.repository.forks_count), @project.repository.forks_url %>
+      </dd>
+      <% if @project.repository.subscribers_count.present? %>
+        <dt class='col-sm-8'>
           Watchers
-        </td>
-        <td>
-          <strong><%= link_to number_with_delimiter(@project.repository.subscribers_count), @project.repository.watchers_url %></strong>
-        </td>
-      </tr>
-    <% end %>
-    <tr>
-      <td>
+        </dt>
+        <dd class='col-sm-4'>
+          <%= link_to number_with_delimiter(@project.repository.subscribers_count), @project.repository.watchers_url %>
+        </dd>
+      <% end %>
+      <dt class='col-sm-8'>
         Contributors
-      </td>
-      <td>
-        <strong><%= link_to number_to_human(@project.contributions.count), @project.repository.contributors_url %></strong>
-      </td>
-    </tr>
-    <% if @project.repository.size.present? %>
-      <tr>
-        <td>
-          Repo Size:
-        </td>
-        <td>
-          <strong><%= number_to_human_size @project.repository.size*1000 %></strong>
-        </td>
-      </tr>
-    <% end %>
+      </dt>
+      <dd class='col-sm-4'>
+        <%= link_to number_to_human(@project.contributions.count), @project.repository.contributors_url %>
+      </dd>
+      <% if @project.repository.size.present? %>
+        <dt class='col-sm-8'>
+          Size:
```

Comment:
This size label could be a little misleading without mentioning it's the size of the repository, for long running projects the repo will get bigger even if the size of the published package doesn't, a big deal for css/js frontend libraries.